dvc/remote/local/__init__.py

```diff
@@ -564,5 +589,97 @@ def unprotect(path_info):
             RemoteLOCAL._unprotect_file(path)
 
     @staticmethod
-    def protect(path_info):
-        os.chmod(path_info.path, stat.S_IREAD | stat.S_IRGRP | stat.S_IROTH)
+    def protect(path):
+        os.chmod(path, stat.S_IREAD | stat.S_IRGRP | stat.S_IROTH)
+
+    def append_verififcation_suffix(self, string):
+        return string + "." + self.STATUS_VERIFICATION_DIR_SUFFIX
+
+    def get_verification_dir_info(self, checksum):
```

Comment:
```suggestion
    @classmethod
    def _get_verification_dir_info(cls, checksum):
```

Reply:
Initially I accepted this suggestion, but this method uses checksum_to_path_info, which cannot be made into classmethod without  making path_info class variable and checksum_to_path a class method